PurrSoft/PurrSoft.Domain/Entities/Animal.cs

```diff
@@ -0,0 +1,14 @@
+﻿using PurrSoft.Domain.Entities.Enums;
+
+namespace PurrSoft.Domain.Entities;
+
+public class Animal
+{
+    public Guid Id { get; set; }
+    public AnimalType animalType { get; set; }
+    public string? Name { get; set; }
+    public int YearOfBirth { get; set; }
+    public string? Gender { get; set; }
+    public Boolean Sterilized { get; set; }
+    public string? ImageUrl { get; set; }
```

Comment:
Would it make sense to also add a Bool whether the animal is alive? We may need to keep histories of registered animals and their Fosters. Also, should we keep some flag if the animal is Adopted or not?